features/environment.py

```diff
@@ -649,31 +649,36 @@ def build_debs_from_dev_instance(context: Context, series: str) -> "List[str]":
         logging.info(
             "--- Launching vm to build ubuntu-advantage*debs from local source"
         )
-        build_container_name = (
-            "ubuntu-behave-image-pre-build-%s-" % series + time_suffix
-        )
-
-        cloud_manager = context.config.cloud_manager
-        if "pro" in context.config.machine_type:
-            user_data = USERDATA_BLOCK_AUTO_ATTACH_IMG
-        else:
-            user_data = ""
-        inst = cloud_manager.launch(
-            instance_name=build_container_name,
-            series=series,
-            user_data=user_data,
-        )
+        chroot_name = "{}-amd64".format(series)
+
+        try:
```

Comment:
Don't know if we need all of this code to check the existence of chroots or even if this is the best approach to check it

Reply:
sbuild tries a few schroot names to figure out the correct one:

> If not specified, the default is the first of schroot  name  or  alias  that matches  $distribution-$arch-sbuild, $distribution-sbuild, $distribution-$arch or $distribution that exists.

so the `"{}-amd64".format(series)` is a bit strict. I'd take the simplest approach: assume that an appropriate schroot exists (= no check), and let sbuild (and therefore the build) fails if it doesn't. Basically I'd treat the "missing schroot" case as we treat build failures in general. Wdyt?

Reply:
I think that's fair. I will just test it out how the error is shown in case we don't have the chroot environment